src/components/Modals/MintItemsModal/MintItemsModal.tsx

```diff
@@ -139,14 +194,42 @@ export default class MintItemsModal extends React.PureComponent<Props, State> {
                 >
                   {t('global.mint')}
                 </NetworkButton>
-              )}
-            </ModalActions>
-            {error ? (
-              <Row className="error" align="right">
-                <p className="danger-text">{error}</p>
               </Row>
-            ) : null}
-          </Form>
+            </div>
+          ) : (
+            <Form>
+              {isEmpty ? (
+                <div className="empty">{t('mint_items_modal.no_items', { name: collection.name })}</div>
+              ) : (
+                items.map(item => <MintableItem key={item.id} item={item} mints={itemMints[item.id]} onChange={this.handleMintsChange} />)
+              )}
+              {isFull ? null : (
+                <ItemDropdown placeholder={t('mint_items_modal.add_item')} onChange={this.handleAddItems} filter={filterAddableItems} />
+              )}
+              <ModalActions>
+                {isEmpty ? (
+                  <Button secondary fluid onClick={onClose}>
+                    {t('global.cancel')}
+                  </Button>
+                ) : (
+                  <NetworkButton
+                    primary
+                    onClick={() => this.setState({ confirm: true })}
+                    loading={isLoading}
+                    disabled={isDisabled || !!error}
```

Comment:
If I'm not wrong the `!!error` condition shouldn't be here, but in the confirm button, right?